2014/02/21 22:16:52 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/AppIngEgr/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/yii/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/yii/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/AppIng...')
#1 /var/www/yii/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/yii/framework/web/CClientScript.php(510): CAssetManager->publish('/var/www/yii/fr...')
#3 /var/www/yii/framework/web/CClientScript.php(547): CClientScript->getCoreScriptUrl()
#4 /var/www/yii/framework/web/CClientScript.php(348): CClientScript->getPackageBaseUrl('jquery')
#5 /var/www/yii/framework/web/CClientScript.php(208): CClientScript->renderCoreScripts()
#6 /var/www/yii/framework/web/CController.php(351): CClientScript->render('<!DOCTYPE html ...')
#7 /var/www/yii/framework/web/CController.php(788): CController->processOutput('<!DOCTYPE html ...')
#8 /var/www/AppIngEgr/protected/controllers/SiteController.php(98): CController->render('login', Array)
#9 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#10 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#14 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#15 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#16 /var/www/AppIngEgr/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/
---
2014/02/21 22:16:53 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/AppIngEgr/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/yii/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/yii/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/AppIng...')
#1 /var/www/yii/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/yii/framework/web/CClientScript.php(510): CAssetManager->publish('/var/www/yii/fr...')
#3 /var/www/yii/framework/web/CClientScript.php(547): CClientScript->getCoreScriptUrl()
#4 /var/www/yii/framework/web/CClientScript.php(348): CClientScript->getPackageBaseUrl('jquery')
#5 /var/www/yii/framework/web/CClientScript.php(208): CClientScript->renderCoreScripts()
#6 /var/www/yii/framework/web/CController.php(351): CClientScript->render('<!DOCTYPE html ...')
#7 /var/www/yii/framework/web/CController.php(788): CController->processOutput('<!DOCTYPE html ...')
#8 /var/www/AppIngEgr/protected/controllers/SiteController.php(98): CController->render('login', Array)
#9 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#10 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#14 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#15 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#16 /var/www/AppIngEgr/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/21 22:46:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/AppIngEgr/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/AppIngEgr/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=gii/form/index
---
2014/02/21 22:46:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/AppIngEgr/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/AppIngEgr/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=gii/model/index
---
2014/02/22 02:48:38 [error] [php] sha1() expects parameter 1 to be string, array given (/var/www/AppIngEgr/protected/controllers/UsuariosController.php:72)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/yii/framework/web/filters/CFilterChain.php(133): UsuariosController->runAction()
#2 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#3 /var/www/yii/framework/web/CController.php(1145): CAccessControlFilter->filter()
#4 /var/www/yii/framework/web/filters/CInlineFilter.php(58): UsuariosController->filterAccessControl()
#5 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#6 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#7 /var/www/yii/framework/web/CController.php(265): UsuariosController->runActionWithFilters()
#8 /var/www/yii/framework/web/CWebApplication.php(282): UsuariosController->run()
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#10 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#11 /var/www/AppIngEgr/index.php(13): CWebApplication->run()
REQUEST_URI=/AppIngEgr/index.php?r=usuarios/create
in /var/www/AppIngEgr/protected/controllers/UsuariosController.php (72)
in /var/www/AppIngEgr/index.php (13)
2014/02/22 02:58:21 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('usuarios/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=usuarios/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=usuarios/index
---
2014/02/22 02:58:22 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('usuarios/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=usuarios/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=usuarios/admin
---
2014/02/22 02:59:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('usuarios/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=usuarios/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=usuarios/create
---
2014/02/22 02:59:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('usuarios/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=usuarios/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=usuarios/admin
---
2014/02/22 03:16:13 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('egresos/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=egresos/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=egresos/create
---
2014/02/22 03:16:14 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in /var/www/yii/framework/web/auth/CAccessControlFilter.php:182
Stack trace:
#0 /var/www/yii/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /var/www/yii/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('egresos/admin')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=egresos/admin
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=egresos/admin
---
2014/02/23 23:42:28 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser._id" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/web/auth/CWebUser.php(146): CComponent->__get('_id')
#1 /var/www/AppIngEgr/protected/controllers/EgresosController.php(69): CWebUser->__get('_id')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): EgresosController->actionCreate()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#10 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('create')
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('egresos/create')
#13 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#14 /var/www/AppIngEgr/index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/AppIngEgr/index.php?r=egresos/create
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/23 23:58:35 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.rememberMe" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/AppIngEgr/protected/components/UserIdentity.php(37): CComponent->__get('rememberMe')
#1 /var/www/AppIngEgr/protected/models/LoginForm.php(52): UserIdentity->authenticate()
#2 /var/www/yii/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/yii/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/yii/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#13 /var/www/AppIngEgr/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/24 00:03:01 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.rememberMe" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/AppIngEgr/protected/components/UserIdentity.php(38): CComponent->__get('rememberMe')
#1 /var/www/AppIngEgr/protected/models/LoginForm.php(52): UserIdentity->authenticate()
#2 /var/www/yii/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 /var/www/yii/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /var/www/yii/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): CModel->validate()
#6 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#7 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#9 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#11 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#12 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#13 /var/www/AppIngEgr/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/24 00:06:43 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.id_usuario" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/yii/framework/web/auth/CWebUser.php(160): CComponent->__set('id_usuario', NULL)
#1 /var/www/AppIngEgr/protected/models/LoginForm.php(72): CWebUser->__set('id_usuario', NULL)
#2 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): LoginForm->login()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/24 00:14:19 [error] [exception.CException] exception 'CException' with message 'UserIdentity and its behaviors do not have a method or closure named "getUser".' in /var/www/yii/framework/base/CComponent.php:266
Stack trace:
#0 /var/www/AppIngEgr/protected/components/WebUser.php(18): CComponent->__call('getUser', Array)
#1 /var/www/AppIngEgr/protected/components/WebUser.php(18): UserIdentity->getUser()
#2 /var/www/AppIngEgr/protected/models/LoginForm.php(71): WebUser->login(Object(UserIdentity), 0)
#3 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): LoginForm->login()
#4 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#5 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#9 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#10 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#11 /var/www/AppIngEgr/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/24 00:16:11 [error] [exception.CException] exception 'CException' with message 'Property "WebUser.id_usuario" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/yii/framework/web/auth/CWebUser.php(160): CComponent->__set('id_usuario', NULL)
#1 /var/www/AppIngEgr/protected/models/LoginForm.php(72): CWebUser->__set('id_usuario', NULL)
#2 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): LoginForm->login()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#4 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#8 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#9 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#10 /var/www/AppIngEgr/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
2014/02/24 00:23:09 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.id_usuario" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/AppIngEgr/protected/models/LoginForm.php(72): CComponent->__get('id_usuario')
#1 /var/www/AppIngEgr/protected/controllers/SiteController.php(94): LoginForm->login()
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('login')
#7 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#8 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#9 /var/www/AppIngEgr/index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/AppIngEgr/index.php?r=site/login
HTTP_REFERER=http://localhost/AppIngEgr/index.php?r=site/login
---
